docs/install/Fedora.md

```diff
@@ -14,7 +14,7 @@ dnf install atomic
 ##Make
 On Fedora, You need to install required build dependencies
 ```
-yum-builddep atomic
+#yum-builddep atomic
```

Comment:
Your patch is to the fedora.md,  If we have a centos.md, we should put your changes their.